app/src/processing/app/EditorTab.java

```diff
@@ -106,6 +107,8 @@ public EditorTab(Editor editor, SketchFile file, String contents)
     file.setStorage(this);
     applyPreferences();
     add(scrollPane, BorderLayout.CENTER);
+    
+    textarea.setupAutoComplete(editor.getSketch(), new FakeCompletionProvider());
```

Comment:
Ok some thoughts about this:

- this call could be commented out, so we can merge this PR without affecting the IDE behavior for now.

- what about adding a method like `Editor.setAutocompletionProvider(CompletionProvider)`? this way we can implement the completion-engine into a tool...

- moreover I think that this call can be inlined (and the method `textarea.setupAutoComplete` removed).